Core/Object Arts/Dolphin/Base/DolphinTest.cls

```diff
@@ -77,6 +77,14 @@ runKnownBug: aString test: aSymbol
 	#todo.	"Make this controllable by an option or class var"
 	^false!
 
+setUp
+
```

Comment:
This seems an inappropriate change to make in the base class of all the Dolphin tests, many of which are completely unrelated to UI activity.